packages/office-ui-fabric-react/src/common/DocPage.types.ts

```diff
@@ -99,6 +99,9 @@ export interface IDocPageProps {
   /** Best practice as markdown string */
   bestPractices?: string;
 
+  /** Feedback section displays recent GitHub issues and provides link to new issue page */
+  feedback?: string;
```

Comment:
What do you pass into this prop as a string? It isn't apparent from the jsdoc. Is it a URL? A labelling text of some sort?

Reply:
I am hoping this to be a query or a list of tags. Is this right?